.github/workflows/regression.yml

```diff
@@ -26,6 +26,13 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+      with:
+        fetch-depth: 2
```

Comment:
Why is this required?

Reply:
Because with default depth (1), you only get the merge commit, and its relation with last tag is impossible to find